pset-iterators.js

```diff
@@ -161,6 +216,13 @@ console.log('Problem 6:')
 
 // Add your code below this line
 
+const firstRepublican = presidents.find(function (president) {
+  return president.party === 'Republican'
+})
+
+console.log(firstRepublican.president)
```

Comment:
@thetripler Great Job here using the find method to return the first president of the Republican party. the `find `method is a good tool for this use case seeing it return the first occurrence of the criteria you give it ex.` return president.party === 'Republican'`.

Take a look at another approach [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L175)